### PR TITLE
docs: add angular docs

### LINKED ISSUE #319: Add @reatom/angular docs
@arswarog already make it (npmjs.org/package/@reatom/angular) 👍
Now we need to add it to the docs.

### PR DESCRIPTION
**Pull request by ilyaryabchinski**
closes #319

### CODE REVIEW

File: docs/packages/angular.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+## Api
+
+### useAtom
+
+Makes Observables from atom
+
+#### Basic (useAtom)
+
+```ts
+const value$ = useAtom(atom)
```
Comment (artalar) [inline docs/packages/angular.md:37]:
```suggestion
const user$ = useAtom(userAtom)
```

File: docs/packages/angular.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+```
+
+#### Depended value by path
+
+```ts
+const value$ = useAtom(atom, 'value')
```
Comment (artalar) [inline docs/packages/angular.md:43]:
```suggestion
const userName$ = useAtom(userAtom, 'name')
```

File: docs/packages/angular.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+Angular bindings package for [Reatom](https://github.com/artalar/reatom) store.
+
+[![npm](https://img.shields.io/npm/v/@reatom/angular?style=flat-square)](https://www.npmjs.com/package/@reatom/angular)
+![npm type definitions](https://img.shields.io/npm/types/@reatom/angular?style=flat-square)
+[![npm bundle size](https://img.shields.io/bundlephobia/minzip/@reatom/angular?style=flat-square)](https://bundlephobia.com/result?p=@reatom/angular)
+![GitHub](https://img.shields.io/github/license/artalar/reatom?style=flat-square)
+
+[Open in docs](https://reatom.js.org/#/packages/angular)
+
+> Reatom is **declarative** and **reactive** state manager, designed for both simple and complex applications. See [docs](https://reatom.js.org/).
```
Comment (artalar) [inline docs/packages/angular.md:12]:
```suggestion

```

Comment (ilyaryabchinski) [reply docs/packages/angular.md]: Thanks for the requested changes 👍
Now it seems to be much more clear and definitive

**Pull request merged**